.ci/scripts/release-manager.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# This script is executed by the daily release artifact stage.
+# It requires the below environment variables:
+# - BRANCH_NAME
+# - VAULT_ADDR
+# - VAULT_ROLE_ID
+# - VAULT_SECRET_ID
+#
+set -uexo pipefail
+
+source /usr/local/bin/bash_standard_lib.sh
+
+# set required permissions on artifacts and directory
+chmod -R a+r build/*
+chmod -R a+w build
```

Comment:
We won't need this if the report dependencies to be consumed by the Unified Release is stored in the build/distributions folder

```suggestion
chmod -R a+r build/distributions/*
chmod -R a+w build/distributions
```